packages/react/src/components/Combobox/Combobox.tsx

```diff
@@ -60,10 +60,14 @@ const defaultAutoCompleteMatches = (inputValue: string, value: string) => {
   return value.toLowerCase().includes(inputValue.toLowerCase());
 };
 
-const ComboboxNoResults = (): JSX.Element => {
+const ComboboxNoResults = ({
+  children
+}: {
+  children?: React.ReactNode;
+}): JSX.Element => {
   return (
-    <div className="ComboboxListbox__empty" aria-live="polite">
-      No results found.
+    <div className="ComboboxListbox__empty" role="alert" aria-live="polite">
```

Comment:
This was the best combination of roles that I could find that would work across the board. Unfortunately for NVDA/Jaws both prepend the announcement with "ALERT" but I could not find a combination with `aria-live`, `aria-atomic`, or `aria-relevant` that would work across the board.